thoth/storages/data/alembic/versions/917d5fccf7b9_add_source_type_column_to_adviser_run_.py

```diff
@@ -0,0 +1,72 @@
+"""Add source type column to adviser run table
+
+Revision ID: 917d5fccf7b9
+Revises: 7c261487d4d4
+Create Date: 2020-06-26 20:43:49.555133+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '917d5fccf7b9'
+down_revision = '7c261487d4d4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('adviser_run', sa.Column('source_type', sa.Text(), nullable=True))
+    op.create_foreign_key(None, 'adviser_run', 'external_software_environment', ['external_run_software_environment_id'], ['id'], ondelete='CASCADE')
```

Comment:
@fridex should I delete all this and add only - `op.add_column('adviser_run', sa.Column('source_type', sa.Text(), nullable=True))`
These were auto-generated, however the initialize_schema() fails..

Reply:
Yes, these should not be part of this migration. I wonder why they were generated by alembic. Was your schema up2date without any manual changes on the database side?

Reply:
> Yes, these should not be part of this migration. I wonder why they were generated by alembic. Was your schema up2date without any manual changes on the database side?

I took a fresh backup and did a schema update, I can try again :)